Source/JNA/waffle-jna/src/main/java/waffle/servlet/NegotiateSecurityFilter.java

```diff
@@ -106,6 +112,20 @@ public void doFilter(final ServletRequest sreq, final ServletResponse sres, fina
         NegotiateSecurityFilter.LOGGER.debug("{} {}, contentlength: {}", request.getMethod(), request.getRequestURI(),
                 Integer.valueOf(request.getContentLength()));
 
+        // If we are not in a windows environment, resume filter chain
+        if (!NegotiateSecurityFilter.isWindows()) {
```

Comment:
I'm late to this PR, sorry, but what's the scenario for this? Why would one ever run this filter on a non-windows system? And if that system supports SSO then why would this be skipping it?

Reply:
I can see using vintela and waffle with both activated.  This would short ciruit one vs the other allowing them to both remain activated without having to comment one or the other filter out.  There are other means to achieve the same but I think trying to do it centrally while waffle doesn't support *nix seems appropriate.  I did downgrade the log to debug in this case as I felt info was too high.  But then again I'm imagining this the way I'm looking to leverage the feature so not sure of requesters ambitions here :)